modules/rtticketedit.php

```diff
@@ -133,6 +133,14 @@
                 $SESSION->redirect('?m=rtqueueview'
                     . ($SESSION->is_set('backid') ? '#' . $SESSION->get('backid') : ''));
                 break;
+            case 'resetpriority':
+                $ticket = $LMS->GetTicketContents($id);
+                if ($ticket['priority'] != 0) {
```

Comment:
A to nie ma stałej w liście priorytetów?

Reply:
Poprawiłem we wszystkich wystąpieniach.